bw2io/export/excel.py

```diff
@@ -185,18 +231,47 @@ def lci_matrices_to_excel(database_name, include_descendants=True):
 
 
 def write_lci_excel(database_name, objs=None, sections=None, dirpath=None):
-    """Export database `database_name` to an Excel spreadsheet.
+    # """Export database `database_name` to an Excel spreadsheet.
```

Comment:
...same question here: [why are you keeping the old docstrings in and commenting them out?](https://github.com/brightway-lca/brightway2-io/pull/169/files#r1141269894)